autoPyTorch/datasets/resampling_strategy.py

```diff
@@ -222,3 +241,48 @@ def get_cross_validators(cls, *cross_val_types: CrossValTypes) -> Dict[str, Cros
             for cross_val_type in cross_val_types
         }
         return cross_validators
+
+
+class NoResamplingFuncs():
```

Comment:
It was hard for me to understand what No resampling means.
`NoSplitsFunc` is way much easier to understand in general, because resampling strategy is not a well-used word.